staff/joel-molero/mailchimp/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mailchimp</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="topbar">
+            <div class="mailchimplogo">
+                <p><img src="../mailchimp/images/img/main_logo.png" alt=""></p>
+            </div>
+            <div class="button">
+                <button class="signupfree">Sign Up Free</button>
+            </div>
+            <div class="mainnav">
+                <nav class="topnav">
+                    <ul>
+                        <li class="li"><a class="link" href="fakeurl" id="pricing">Pricing</a></li>
+                        <li class="li2"><a class="link" href="fakeurls" id="features">Features</a></li>
+                        <li class="li2"><a class="link" href="fakeurlss" id="resources">Resources</a></li>
+                        <li class="li2"><a class="link" href="fakeurlsss" id="features">Features</a></li>
+                        <li class="li2"><a class="link" href="fakeurlssss" id="blog">Support</a></li>
+                        <li class="li3"><form id="search">
+                            <input type="text" placeholder="Search"></form>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+        <div class="title-img">
+                <div class="img">
+                    <img class="monkey" src="./images/easy-email-newsletters-banner-png8.png" alt="chimp">
```

Comment:
To have it adaptive:
> height:81%;
> padding-left:40%
>

Reply:
That kinda messes up the image and the whole style of the front page, plus the bottom section is not responsive either, so it just overlays with the text. Might be a good idea to implement on future versions though.